React/React18/protected/package.json

```diff
@@ -41,8 +41,8 @@
     ]
   },
   "devDependencies": {
-    "@preemptive/jsdefender-core": "file:assets/preemptive-jsdefender-core-{version}.tgz",
-    "@preemptive/jsdefender-webpack-plugin": "file:assets/preemptive-jsdefender-webpack-plugin-{version}.tgz",
+    "@preemptive/jsdefender-core": "~2.5.0-next.0",
```

Comment:
I don't think this is the proper way for implementation, we need to keep "latest", please roll-back

Reply:
Sorry, my bad, this was a wrong chance